new/LIT6193HomiliaryPassions.xml

```diff
@@ -0,0 +1,155 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6193HomiliaryPassions" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Homiliary on the Passions of Christ</title>
+                <title xml:lang="gez" xml:id="t2">ድርሳን፡ ማኅያዊ፡</title>
+
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The work is different from <ref type="work" corresp="LIT1291Dersan"/>.</p>
```

Comment:
relation can be created for this; the record LIT1291Dersan should also be corrected accordingly (it is written there 

```
It is not entirely clear from the catalogues whether the work contained in
<ref type="mss" corresp="BNFet116#ms_i1"/>
is the same as in
<ref type="mss" corresp="Bladd16254#ms_i1"/>
and
<ref type="mss" corresp="BNFabb29"/>
.
```

Reply:
They are different. Ok, I'll clarify.